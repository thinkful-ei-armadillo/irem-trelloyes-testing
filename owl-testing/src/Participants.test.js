import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('<App />', () => {
it('renders without crashing', () => {
  const div = document.createElement('div');

  ReactDOM.render(<Card />, div);

  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="Test card" content ="Card component working"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    })
})