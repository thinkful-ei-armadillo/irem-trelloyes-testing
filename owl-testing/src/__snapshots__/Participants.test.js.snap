// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the UI as expected 1`] = `
<section
  className="List"
>
  <div
    className="List-participants"
  >
    <div
      className="Participant"
    >
      <img
        alt="headshot"
        src="https://ia.media-imdb.com/images/M/MV5BMTY5NzE3NzU3MF5BMl5BanBnXkFtZTgwMjg0NTQ5MDE@._V1_UX214_CR0,0,214,317_AL_.jpg"
      />
      <h3>
        Sally Salonga
      </h3>
      <p />
      <p />
    </div>
    <div
      className="Participant"
    >
      <img
        alt="headshot"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
      <h3>
        Joe Turner
      </h3>
      <p />
      <p />
    </div>
  </div>
</section>
`;
